### ISSUE TITLE
Backstabbing wrongfully possible

### ISSUE THREAD

**Issue by futscdav** [elysium-project/server#3110]
It is possible to use backstab on mobs that are directly attacking you by running straight through them while spamming backstab at the same time. That is not the behavior that was on the retail server in those times (or now)

**Comment by sadfx** [comment]
Strafestabbing was possible.

**Comment by futscdav** [comment]
no, it was not possible against a mob that was directly attacking you due to how it was checked (client side)

**Comment by sadfx** [comment]
It was possible. Move along.

**Comment by futscdav** [comment]
unless you provide hard evidence of it, i will firmly stand behind this being a bug. certainly in the way it is implemented here.

**Comment by taurenrogue** [comment]
> unless you provide hard evidence of it, i will firmly stand behind this being a bug. certainly in the way it is implemented here.
You are the one making a bug report without a source not the comment guy
